index.css

```diff
@@ -0,0 +1,352 @@
+  z-index: 1;
+}
+.cross-image {
+  max-height: 30rem;
+  object-fit: fill;
+}
+.exit-button:hover {
+  background: cadetblue;
+}
+.card-img img {
+  width: 100%;
+  height: 30rem;
+}
+.cart-button {
+  padding: 0.8rem;
+  /* width:10rem;
+          height:3rem; */
+
+  border: none;
+  outline: none;
+  background: black;
+  color: cornsilk;
+}
+.cart {
+  margin: 1rem;
+  padding: 1rem;
+}
+
+.card-body,
+.original-price,
+.discount-price,
+.price {
+  font-size: 2rem;
+}
+
+/* button component */
+
+.button-container-button {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
+  display: inline-block;
+  width: 15rem;
+  font-weight: 800;
+  font-size: 1.8rem;
+  border: none;
+  border-radius: 1rem 1rem 1rem 1rem;
+
+  color: black;
+  font-family: "roboto";
+}
+.primary-button {
+  background-color: #49a0eb;
+}
+.secondary-button {
+  background-color: #63edff;
+  color: black;
+  font-family: "roboto";
+}
+.danger-button {
+  background-color: red;
+  color: white;
+  font-family: "roboto";
+}
+.toggle-button {
+  position: relative;
+
+  width: 16rem;
+  height: 5rem;
+  border: none;
+
+  background-color: #63edff;
+  color: black;
+  font-family: "roboto";
+}
+.icon-badge-number {
+  position: absolute;
+  top: 0;
+  right: -2.1rem;
+  width: 2.2rem;
+  height: 2.2rem;
+  background: #209bab;
+  color: black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+}
+.toggle-badge {
+  width: 16rem;
+  height: 5rem;
+  position: relative;
+}
+.hide {
+  display: none;
+}
+.floating-action-button {
+  margin-top: 2rem;
+  width: 5rem;
+  height: 5rem;
+  border-radius: 50%;
+  border: none;
+  background-color: #49a0eb;
+  z-index: 1;
+}
+.floating-action-button-small {
+  margin-top: 2rem;
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  border: none;
+  background-color: #49a0eb;
+}
+.floating-action-button:hover,
+.floating-action-button-small:hover {
+  background-color: #63edff;
+}
+.floating-action-img {
+  width: 100%;
+  object-fit: contain;
+  align-items: center;
+}
+
+/* badge */
+.notification {
+  position: relative;
+  width: 5rem;
+  height: 5rem;
+}
+.icon-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 4rem;
+  height: 4rem;
+  color: #333333;
+  background: lavenderblush;
+  border: none;
+  outline: none;
+  border-radius: 50%;
+}
+.icon-img {
+  width: 100%;
+}
+.icon-badge-number {
+  position: absolute;
+  top: 0;
+  right: 0rem;
+  width: 1.5rem;
+  height: 1.5rem;
+  background: lightblue;
+  color: black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+}
+
+/* sidebar */
+.side-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 25rem;
+  background-color: grey;
+  height: 100vh;
+}
+.side-bar h1 {
+  margin: 1rem;
+}
+.side-bar h2 {
+  font-size: 1.7rem;
+  margin: 1rem;
+  border-bottom: 0.1rem solid black;
```

Comment:
Suggestion: Try using px for smaller values of rem. Because if the value is small, it is almost not noticeable if we change the root font-size.